src/main/java/seedu/eduke8/note/NoteList.java

```diff
@@ -17,24 +17,46 @@ public NoteList(ArrayList<Displayable> notes) {
         this.notes = notes;
     }
 
+    /**
+     * Adds a Displayable object into the NoteList.
+     *
+     * @param note A note, which is an object that implements the Displayable interface.
+     */
     @Override
     public void add(Displayable note) {
         notes.add(note);
     }
 
+    /**
+     * Deletes a Displayable object (specified with the provided int index) from the NoteList.
+     *
+     * @param index An index, which indicates which note in the NoteList to is to be deleted.
+     */
     @Override
     public Displayable delete(int index) {
```

Comment:
Sorry Joel, what I meant was should there be a @return here?

Reply:
Good catch! Fixed it. Thanks!